ui/app/AppLayouts/Wallet/controls/AssetDelegate.qml

```diff
@@ -48,8 +49,8 @@ Item {
         font.pixelSize: 15
     }
     StyledText {
-        id: assetBalance
-        text: Utils.toLocaleString(balance, localAppSettings.locale) + " " + symbol.toUpperCase()
+        id: assetAddress
+        text: Utils.toLocaleString(currencyBalance.toFixed(2), locale) + " " + assetDelegate.currency.toUpperCase()
```

Comment:
Are these three changes correct? We're changing what was the `assetFullTokenName` to `assetBalance` and what was `assetSymbol` to `assetFullTokenName` etc.

This looks like a change in logic. Is this intended?

Reply:
I just looked at the designs for wallet2 and thought updating, but reverted the change now.

Reply:
Thanks!